src/post/workshed.ts

```diff
@@ -74,20 +79,21 @@ const worksheds = [
     },
     action: () => {
       if (!TrainSet.canConfigure()) return;
+      if (globalOptions.ascend && estimatedTurns() <= 40) return;
```

Comment:
Oh are you not allowed to reconfigure after ascending if 40 turns have not passed since the last ascension?

Reply:
Not quite--your configuration is saved upon ascending, so ascenders will want the option to configure it themselves, ascend, and then have both the configuration they want and the ability to reconfigure at will.

Reply:
idk if I agree with this since this is technically meat-suboptimal, and only applies to users who would run trainset immediately after ascension, or specifically time their counter such that they swap in their trainset at an opportune moment.
But at the same time this only comes at a cost of, what, 1.9k? I can probably live with that; I just don't know whether this is in the spirit of things (but I also do not want to continually tag on more and more custom prefs/args which would make managing garbo way harder in the future)

Reply:
I agree re: args and prefs, but also I feel like in the competition between "maybe 2k for loopers, if the numbers work out like that" and "not locking Actual Ascenders into a suboptimal state" I care a lot more about the latter.